src/infra/repos/postgresql/tag-repository.ts

```diff
@@ -37,4 +37,15 @@ export class TagRepository implements AddTagRepository, LoadTagByIdRepository, U
     })
     return tag
   }
+
+  async loadAll (): Promise<LoadTagsRepository.Result> {
+    const tags = await db.tag.findMany({
+      select: {
+        id: true,
+        name: true,
+        color: true
+      }
+    })
```

Comment:
tira o select já que você ta retornando tudo
```suggestion
    const tags = await db.tag.findMany()
```